src/main/java/com/google/gcp_variant_transforms/library/VariantToBqUtilsImpl.java

```diff
@@ -199,29 +196,30 @@ public void addFormatAndPhaseSet(TableRow row, Genotype genotype, VCFHeader vcfH
         continue; // We will set GT field in a separate "genotype" field in BQ row.
       }
       if (genotype.hasAnyAttribute(attrName)) {
+        String sanitizedFieldName = SchemaUtils.getSanitizedFieldName(attrName);
         Object value = genotype.getAnyAttribute(attrName);
-        VCFFormatHeaderLine formatMetadata = vcfHeader.getFormatHeaderLine(attrName);
-        VCFHeaderLineType formatType = formatMetadata.getType();
+        VCFHeaderLineType formatType = formatHeaderLine.getType();
         if (attrName.equals(VCFConstants.GENOTYPE_ALLELE_DEPTHS) ||
             attrName.equals(VCFConstants.DEPTH_KEY) ||
             attrName.equals(VCFConstants.GENOTYPE_QUALITY_KEY) ||
             attrName.equals(VCFConstants.GENOTYPE_PL_KEY)) {
           // These four field values have been pre-processed in Genotype.
-          row.set(attrName, value);
+          row.set(sanitizedFieldName, value);
         } else if (attrName.equals(VCFConstants.PHASE_SET_KEY)) {
           String phaseSetValue = genotype.getAnyAttribute(VCFConstants.PHASE_SET_KEY).toString();
           phaseSet = replaceMissingWithNull(phaseSetValue);
         } else {
           // The rest of fields need to be converted to the right type as VCFHeader specifies.
-          VCFHeaderLineCount formatCountType = formatMetadata.getCountType();
+          VCFHeaderLineCount formatCountType = formatHeaderLine.getCountType();
           if (formatCountType == VCFHeaderLineCount.INTEGER) {
-            row.set(attrName, convertToDefinedType(genotype.getAnyAttribute(attrName),
-                formatType, formatMetadata.getCount()));
+            row.set(sanitizedFieldName, convertToDefinedType(genotype.getAnyAttribute(attrName),
+                formatType, formatHeaderLine.getCount()));
           } else {
             // If field number in the VCFHeader is ".", should pass a default count and do not
-            // check if count is equal to the size of value.
-            row.set(attrName, convertToDefinedType(genotype.getAnyAttribute(attrName),
-                formatType, Constants.DEFAULT_FIELD_COUNT));
+            // check if count is equal to the size of value. And the field should be repeated
+            // (list of values).
+            row.set(sanitizedFieldName, convertToDefinedType(genotype.getAnyAttribute(attrName),
+                formatType, Constants.DEFAULT_REPEATED_FIELD_COUNT));
           }
         }
       }
```

Comment:
The following if statement that examines `phaseSet` value can be simpler by:
* Initiate `String phaseSet = Constants.DEFAULT_PHASESET`;
* Remove the if statement and replace it with: `row.set(Constants.ColumnKeyNames.CALLS_PHASESET, phaseSet);`

What do you think?